browser/css/jsdialogs.css

```diff
@@ -308,7 +308,7 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 }
 
 .ui-tabs-content.jsdialog > .ui-content.hidden {
-	display: inherit !important;
+	display: none !important;
```

Comment:
`visibility: hidden` causes element to be hidden but still occupying the space
`display: none` removes the element completely
This is important inside grids where we have specific order of columns and rows

Reply:
we did that to keep the same size of every tab in the dialog so "Next" button in the wizard will be at the same place, so user can just click multiple times in the same place